.buildkite/pipeline.yml

```diff
@@ -194,6 +194,17 @@ steps:
     concurrency: 4
     concurrency_group: 'ruby/unit-tests'
 
+  - label: ':ruby: Ruby 3.0 unit tests'
+    timeout_in_minutes: 30
+    plugins:
+      docker-compose#v3.1.0:
+        run: ruby-unit-tests
+        use-aliases: true
+    env:
+      RUBY_TEST_VERSION: "3.0.0-preview2"
```

Comment:
TODO: this version needs changing to 3.0 when it's released & there's a tag on docker hub